pace-util/pace/util/quantity.py

```diff
@@ -284,6 +284,13 @@ def __init__(
         else:
             extent = tuple(extent)
 
+        mask = tuple(
+            [
+                any(dim in coord_dims for dim in dims)
+                for coord_dims in [constants.X_DIMS, constants.Y_DIMS, constants.Z_DIMS]
+            ]
+        )
```

Comment:
optional nit: You could put this inside the if block for gt4py_backend is not None, so it only gets computed when needed.

Reply:
Good idea, done!